src/avfoundation.cs

```diff
@@ -10317,11 +10335,9 @@ interface AVPlayer {
 		[Notification]
 		NSString AvailableHdrModesDidChangeNotification { get; }
 
-		// From AVPlayer (AVPlayerVideoDecoderGPUSupport) Category
-
-		[NoWatch, NoTV, NoiOS, Mac (10,13,4)]
+		[NoWatch, NoTV, NoiOS, Mac (10, 13)]
 		[Export ("preferredVideoDecoderGPURegistryID")]
-		ulong PreferredVideoDecoderGpuRegistryId { get; set; }
+		ulong PreferredVideoDecoderGPURegistryId { get; set; }
```

Comment:
double bad - `GPU` in caps

Reply:
and I don't have `PreferredVideoDecoderGPURegistryId` in `master` ?!? where does that comes from ?

Reply:
reverting.